composer.json

```
@@ -12,10 +12,10 @@
     },
     "require": {
         "php": ">=7.1.0",
-        "drupal/core": "~8.7"
+        "drupal/core": "~8.7 || ^9.0"
     },
     "require-dev": {
-        "drupal/coder": "~8.2"
+        "drupal/coder": "~8.2 | ^8.3"
```

@heddn, shouldn't we use this:

```suggestion
        "drupal/coder": "^8.2"
```

---

Yeah, better.